main_rate: 10.0 # [Hz]
swarming_rate: 10.0 # [Hz]

swarming:
  max_velocity: 0.75 # (m/s)
  visibility_radius: 10.0 # (m)

uav_names: [
  "uav1",
  "uav2",
  "uav3",
  "uav4",
  "uav5",
  "uav6",
  "uav7",
  "uav8",
  "uav9",
  "uav10",
  "uav11",
  "uav12",
  "uav13",
  "uav14",
  "uav15",
  "uav16",
  "uav17",
  "uav18",
  "uav19",
  "uav20",
]
